pkg/controllers/dynakube/injection/reconciler.go

```diff
@@ -43,6 +48,7 @@ func NewReconciler(
 	istioClient *istio.Client,
 	fs afero.Afero,
 	dynakube *dynatracev1beta1.DynaKube,
+	scheme *runtime.Scheme,
```

Comment:
have the order be 
```
	client client.Client,
	apiReader client.Reader,
	scheme *runtime.Scheme,
	...
```

(we do this "everywhere")

also, fun fact `fs afero.Afero` is not needed, its a leftover in the `versionReconciler` (I encountered it in [my pr](https://github.com/Dynatrace/dynatrace-operator/pull/2781))

Reply:
basically done in https://github.com/Dynatrace/dynatrace-operator/pull/2777/commits/39d3778db07cc9d89ba637c88bb091975411c30b

BUT i was not sure, should `fs` be removed as it is not needed anymore? Because I did not do that yet.